src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/TermQueryExplainIT.java

```diff
@@ -275,6 +278,65 @@ public void testJoinAliasMissing() throws IOException {
 
     }
 
+    @Test
+    public void testNestedSingleConditionAllFields() throws IOException {
+        String result = explainQuery(String.format(Locale.ROOT,
```

Comment:
we have `StringUtils.format` to avoid specifying the locale, but I think Andy wants us to avoid using format for straightforward queries. Up to you which one you pick, but if you keep format, use ours.

Reply:
Will be inlining index name and getting rid of string format entirely a suggested by @galkk